### PR TITLE
Add Pagerduty severity for workflows

### PR DESCRIPTION
**Pull request by haydentherapper**
Staging workflows will be low priority (warning), and production will be high priority (error). Rate limiting will be low priority since it's not something critical to respond to.

Ref https://github.com/sigstore/public-good-instance/issues/1568

### CODE REVIEW

File: .github/workflows/ctlog-sth.yml [hunk 1]
```diff
@@ -54,6 +54,11 @@ jobs:
           fi
           if ! go run ./prober/ctlog/ctlog-sth.go --env ${{ matrix.env }} --shard ${{ matrix.shard }} ; then
             echo "probe_status=failed" >> $GITHUB_OUTPUT
+            if [[ "${{ matrix.env }}" == "production" ]]; then
+              echo "severity=error" >> $GITHUB_OUTPUT
```
Comment (cpanato) [inline .github/workflows/ctlog-sth.yml:58]:
```suggestion
              echo "severity=error" >> "$GITHUB_OUTPUT"
```

nit

**Review by bobcallaway — APPROVED** [review]

**Comment by haydentherapper** [comment]
I can use Event Orchestration rules in Pagerduty to dynamically change severity for staging events over the weekend, so this PR should be good to go.

**Comment by haydentherapper** [comment]
Don't think I need this, going to use `group` instead.

**Review by bobcallaway — APPROVED** [review]

**Comment by haydentherapper** [comment]
Not needed, we'll use `group` to differentiate events in PG

**Pull request closed without merging**